code/game/machinery/turrets.dm

```diff
@@ -645,6 +650,12 @@
 			if(M.stat || M.lying || (M in exclude))
 				continue
 			pos_targets += M
+		for(var/obj/effect/blob/B in oview(scan_range, src))
+			pos_targets += B
+		for(var/mob/living/simple_animal/hostile/blobspore/BS in oview(scan_range,src))
+			if(BS.stat || BS.lying || (BS in exclude))
```

Comment:
.gcDestroyed too

Reply:
It really should not be in `oview()` if it's been `qdel()`ed

Reply:
true dat